test/helpers.rb

```diff
@@ -59,6 +59,10 @@ def filechecks(except = nil)
     checks
   end
 
+  def array_sorted?(array)
+    array.each_cons(2).all? { |i, j| (i <=> j) <= 0 }
```

Comment:
The chaining of `<=>` and `<=` is a bit redundant, isn't it? `i <= j` will have the exact same effect. (But read on in the next code comment before you address this.)